packages/drivers/redux-little-router/misc/test-suite.js

```diff
@@ -0,0 +1,244 @@
+/* eslint-env jest */
+import { push } from 'redux-little-router'
+import { createStore as krmlCreateStore, when } from 'k-ramel'
+import omit from 'lodash/omit'
+
+export default (driver) => {
+  /* create k-ramel store with redux-little-router driver */
+  const createStore = (description, options, routes) => {
+    const router = driver({ routes }, state => state.router)
```

Comment:
**2 interrogations :**
1. It may be better to extract router creation from the driver : 
```js
const router = driver(routerForBrowser({ routes }), state => state.router)
```

2. I don't if it's better to put  state `selector` here or at the driver declaration

What do you think about that ?

Reply:
1. k-ramel way of doing this would be: 
  a. Make it simple for 90% of use cases: `const router = driver(routes)`
  b. Make it possible to load custom configuration: `const router = driver(routes, router)`

2. Let it in one line for now.

---

Maybe we should refactor the driver so drivers returns this interface everytime:
 - getEnhancer: returns enhancer
 - getReducer: returns { path, reducer }
 - getDriver: returns driver

And we use them like that:
```js
export default createStore(
  description,
  {
    drivers: [
      http(), 
      form(state => state.form),
      reduxLittleRouter(routes, state => state.router, routerImpl),
    ]
  }
)
```

And `k-ramel` use the returned interface to configurate the store where it needs to be, enhancer would be pushed the same order as driver declaration.

But this is in an other PR/Issue

Reply:
#103